DiscordBee.cs

```diff
@@ -136,6 +138,27 @@ public void ReceiveNotification(string sourceFileUrl, NotificationType type)
             _updateTimer.Start();
           }
           break;
+        case NotificationType.PlayingTracksChanged:
+          UploadQueuedTracks();
+          break;
+      }
+    }
+
+    private void UploadQueuedTracks()
+    {
+      for (int i = 0; i < 4; i++)
+      {
+        int nextPlayingIndex = _mbApiInterface.NowPlayingList_GetCurrentIndex();
+        string fileUrl = _mbApiInterface.NowPlayingList_GetListFileUrl(nextPlayingIndex + 1 + i);
+        if (fileUrl == null || fileUrl == "")
```

Comment:
There is https://docs.microsoft.com/en-us/dotnet/api/system.string.isnullorempty?view=net-6.0
Also please always add braces to ifs even for oneliners